docs/installation.md

```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
-:::
+### llvm 📜
 
-#### Rust 🦀
+Install llvm. For instance, on MacOS, run:
 
-> `jstz` requires a specific release of Rust. The version of Rust required is specified in the `rust-toolchain` file.
+```sh
+brew install llvm
+```
 
-Install the [Rust](https://rustup.rs/) toolchain:
+#### Nix 📦
+
+Both `jstz` and Octez are packaged with Nix, a package manager and system configuration tool that makes building from sources easy!
+
+On MacOS, simply run and follow the interactive setup:
 
 ```sh
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+sh <(curl -L https://nixos.org/nix/install)
 ```
 
-Once `rustup` is installed, the build dependencies can be installed with:
+For other operating systems, go to the [Nix download page](https://nixos.org/download.html) and install the _Multi-user installation_ variant of _Nix: the package manager_.
+
+Additionally, to setup nix ensuring nix-flakes are enabled, run:
 
 ```sh
-make build-deps
+nix-shell --experimental-features 'nix-command flakes'
 ```
 
-::: tip
-Using Nix, simply run `nix develop` to enter a shell with all build dependencies
-:::
+#### Rust 🦀
 
-#### Octez 🐙
+`jstz` requires a specific release of Rust. The version of Rust required is specified in the `rust-toolchain` file.
 
-Our sandbox network uses a custom distribution of Octez found [here](https://gitlab.com/tezos/tezos/-/tree/6c0621760ddce94afeff3484d9e8a650d8535f25). See the [Octez docs](https://tezos.gitlab.io/introduction/howtoget.html?highlight=building#compiling-with-make) for instructions on building Octez from source.
+Install the [Rust](https://rustup.rs/) toolchain:
 
-Once Octez has been built, copy the following binaries to `jstz`:
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
 
-- `octez-client`
-- `octez-node`
-- `octez-smart-rollup-node`
-- `octez-smart-rollup-wasm-debugger`
+#### Octez 🐙
 
-::: tip
+`jstz` sandbox network uses Octez and it needs to be built from source (for the time being).
 
-Using Nix, simply execute the following:
+Using Nix, go to the Octez repository cloned [earlier](#cloning-the-repositories) simply execute the following:
 
 ```sh
-# Clone Octez
-git clone git@gitlab.com:tezos/tezos.git
 cd tezos
-# Checkout custom distribution
-git checkout ole@next-gen@floats
-# Build using Nix
 nix-build -j auto
 ```
 
-After Nix successfully builds Octez (it takes a long time 🕣), the Octez binaries will be accessable from `result`.
-:::
+After Nix successfully builds Octez (which takes a long time 🕣), the Octez binaries will be available in the `result/bin/` directory. Copy the following binaries to the `jstz` root direactory:
+
+- octez-client
+- octez-node
+- octez-smart-rollup-node
+- octez-smart-rollup-wasm-debugger
```

Comment:
`octez-smart-rollup-wasm-debugger` should be dropped. Not required for the sandbox and is no-longer a useful tool for us